content/docs/1.6.0/deploy/install/_index.md

```diff
@@ -226,6 +226,8 @@ The command used to install a NFSv4 client differs depending on the Linux distri
   zypper install nfs-client
   ```
 
+- For Talos Linux, [The NFS client is part of the `kubelet` image maintained by the Talos team](https://www.talos.dev/v1.6/kubernetes-guides/configuration/storage/#nfs).
```

Comment:
```suggestion
- For Talos Linux, the NFS client is part of the [`kubelet` image maintained by the Talos team](https://www.talos.dev/v1.6/kubernetes-guides/configuration/storage/#nfs).
```

Reply:
`The NFS client is part of the kubelet image maintained by the Talos team`
is obtained from the linked NFS section. IMO it looks a bit odd just linking the `kubelet image maintained by the Talos team`. Do you have an alternative suggestion?

Reply:
I expanded the link text to include the first part. However, we must use lowercase "t" here because the link text is still part of the sentence.